predicators/ground_truth_models/spot_env/options.py

```diff
@@ -90,6 +113,67 @@ def _grasp_at_pixel_and_maybe_stow_or_dump(
         stow_arm(robot)
 
 
+def _sim_safe_grasp_at_pixel_and_maybe_stow_or_dump(
+        robot: Robot, target_obj: Object, rng: np.random.Generator,
+        timeout: float, retry_grasp_after_fail: bool, do_stow: bool,
+        do_dump: bool) -> None:
+    """Implicitly gets the current image from the hand camera and selects a
```

Comment:
I know this is an unavoidable hack for now, but I do think we can reduce the amount of copied code between here and the function above